mage_ai/services/teams/teams.py

```diff
@@ -0,0 +1,17 @@
+from mage_ai.services.teams.config import TeamsConfig
+import json
+import requests
+
+
+def send_teams_message(config: TeamsConfig, message: str ,title = "Mage Ai Pipeline Run Status Logs",) -> None:
```

Comment:
Can we change the default message to: Mage pipeline run status logs.

Also, please use single quotes

Reply:
Ok Sure

Reply:
Done